### PR TITLE
Add ro.hume.cosmin.ShoppingList

### PR DESCRIPTION
**Pull request by ghfh0**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by nedrichards** [comment]
bot, build ro.hume.cosmin.ShoppingList

File: ro.hume.cosmin.ShoppingList.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+    "app-id" : "ro.hume.cosmin.ShoppingList",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "42",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "shopping-list",
+    "finish-args" : [
+        "--share=network",
```
[the remark this answers is not in the document]
Comment (ghfh0) [reply ro.hume.cosmin.ShoppingList.json]: I don't. I removed it.

File: ro.hume.cosmin.ShoppingList.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    ],
+    "modules" : [
+        {
+            "name" : "shopping-list",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://gitlab.gnome.org/cosmin/shopping-list.git"
```
[the remark this answers is not in the document]
Comment (ghfh0) [reply ro.hume.cosmin.ShoppingList.json]: Oops, I forgot about it. Added now.

**Comment by hfiguiere** [comment]
bot, build ro.hume.cosmin.ShoppingList

**Pull request closed without merging**